test-harness/src/test/java/com/netflix/conductor/tests/integration/End2EndTests.java

```diff
@@ -247,100 +265,4 @@ public void testMetadataWorkflowDefinition() {
         }
     }
 
-    @Test
```

Comment:
These tests have been moved all the way up to the abstract class on behalf of not duplicating code. They are used on `End2EneTests` and `End2EndGrpcTests`